src/lib/builder/board-builder.ts

```diff
@@ -78,7 +69,7 @@ export class BoardBuilderClass
   }
 
   async build(bc: Type.BuildContext): Promise<Type.AnyElement[]> {
-    const required_props = ["width", "height", "center_x", "center_y"]
+    const required_props = ["width", "height", "pcbX", "pcbY"]
```

Comment:
Can you add a quick remapping condition here for backwards compatibility. Eg if (center_x !== undefined) props.pcbX = center_x

Reply:
https://github.com/tscircuit/builder/pull/76/commits/297512eb900697073bcf8bcc9f70802567bd4afb

Reply:
https://github.com/tscircuit/builder/pull/76/commits/f8a5e64cf05b3e37efca3d2a825e353cb581ee93

Reply:
nice, you can use `LegacyBoardProps` as a type to fix the type issue i think

Reply:
I do not see a type like that

Reply:
you'll need to create it yep

Reply:
just throw it up at the top of the file

Reply:
I guess I only need to use it for the setProps prob in that exact one function nowhere else

Reply:
yep!